sdk/core/Cargo.toml

```diff
@@ -34,7 +34,7 @@ async-trait = "0.1.36"
 oauth2 = { version = "4.0.0-alpha.2" }
 
 [dev-dependencies]
-tokio = "0.3"
+tokio = "0.2"
```

Comment:
I got some runtime errors regarding the hyper dependency. I think we may need to wait for hyper to upgrade.

Reply:
- hyper v0.14 https://github.com/hyperium/hyper/issues/2302
- reqwest ? https://github.com/seanmonstar/reqwest/issues/1060

Reply:
I created #60 to track upgrading.